build.gradle

```diff
@@ -5,6 +5,7 @@ buildscript {
   dependencies {
     classpath 'com.android.tools.build:gradle:2.2.2'
     classpath 'com.github.ben-manes:gradle-versions-plugin:0.13.0'
+    classpath 'com.neenbedankt.gradle.plugins:android-apt:1.8'
```

Comment:
Might also be interesting: http://www.littlerobots.nl/blog/Whats-next-for-android-apt/